filebeat/tests/system/test_modules.py

```diff
@@ -95,14 +97,21 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
         objects = [o["_source"] for o in res["hits"]["hits"]]
         assert len(objects) > 0
         for obj in objects:
-            self.assert_fields_are_documented(obj)
-            # assert "error" not in obj  # no parsing errors
-            assert obj["fileset"]["module"] == module
+            assert obj["fileset"]["module"] == module, "expected fileset.module={} but got {}".format(
+                module, obj["fileset"]["module"])
+
+            if not (module == "mysql" and fileset == "slowlog") and not (module == "system" and fileset == "auth"):
+                # TODO: There are errors parsing the test logs from these modules.
+                assert "error" not in obj
+
+            if module != "system" and fileset != "audit":
+                # There are dynamic fields in audit logs that are not documented.
```

Comment:
The main thing I worry is that a dynamic field could overwrite a static field and conflict on the type. Could we perhaps just have one entry with all the fields as a string? Or use the `raw` namespace? Also not sure yet how to deal with it.